contrib/k8s/charts/open-service-broker-azure/README.md

```diff
@@ -101,7 +101,7 @@ Broker chart and their default values.
 | Parameter | Description | Default |
 | --------- | ----------- | ------- |
 | `logLevel` | Log level (options: PANIC, FATAL, ERROR, WARN, INFO, DEBUG). | `"INFO"` |
-| `image.repository` | Docker image location, _without_ the tag. | `"microsoft/azure-service-broker"` |
+| `image.repository` | Docker image location, _without_ the tag. | `"osbapublicacr.azurecr.io/microsoft/azure-service-broker"` |
```

Comment:
If (your choice) you do anything about [this comment](https://github.com/Azure/open-service-broker-azure/pull/699/files#r281196692), this address also needs to change.